tokio/Cargo.toml

```diff
@@ -50,7 +50,7 @@ io-util = ["memchr", "bytes"]
 io-std = []
 macros = ["tokio-macros"]
 net = [
-  "lazy_static",
```

Comment:
It looks like net doesn't use lazy_static (anymore?) so I think it is safe to omit this line completely!

Reply:
Done! :)